From 6005641dece08fe6368d91ca21591e4223e5a50b Mon Sep 17 00:00:00 2001
From: Kalev Lember <klember@redhat.com>
Date: Fri, 17 Jul 2015 19:22:45 +0200
Subject: [PATCH] Avoid registering two main desktop file categories

gnome-shell classic mode puts apps that are registered with more than
one main category under Other. To fix this so that we properly end up
under System, this commit removes Settings category from the desktop
file.
---
 data/gpk-application.desktop.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/gpk-application.desktop.in b/data/gpk-application.desktop.in
index 2efdf70..dac83eb 100644
--- a/data/gpk-application.desktop.in
+++ b/data/gpk-application.desktop.in
@@ -5,7 +5,7 @@ Icon=system-software-install
 Exec=gpk-application
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;System;Settings;PackageManager;
+Categories=GNOME;GTK;System;PackageManager;
 _Keywords=Updates;Upgrade;Sources;Repositories;Preferences;Install;Store;
 NotShowIn=KDE;
 StartupNotify=true
-- 
2.4.3

